app/index.html

```diff
@@ -0,0 +1,675 @@
+
+                <div class="social-menu">
+                    <a class="social-button linkedin" href="https://www.linkedin.com/in/ellie-robinson-63326b133/" target="_blank">
+                        <i class="fab fa-linkedin"></i>
+                    </a>
+                    <a class="social-button github" href="https://github.com/ellie-r" target="_blank">
+                        <i class="fab fa-github"></i>
+                    </a>
+                    <a class="social-button email" href="mailto: ejr.jobs@gmail.com" target="_blank">
+                        <i class="fas fa-envelope-square"></i>
+                    </a>
+                </div>
+
+            </div>
+        </div>
+    </div>
+    <div class="container">
+        <div class="row aboutSection section" id="AboutMe">
+            <div class="col-md-9 text">
+                <p>
+                    I graduated from the University of Exeter in 2019 with a 2:1 in Maths.
+                    After graduating I worked for a while as a Junior Data Analyst where I decided that solving problems
+                    with code was the favourite part of my day and I wanted to pursue it as a career!
+                </p>
+                <p>
+                    I decided to join iO Academy (prev. Mayden Academy),
+                    a 16 week intensive course, to learn all the skills required for the industry.
+                </p>
+                <p>
+                    I like reading sci-fi and fantasy books, binging Netlix shows and playing with my two miniature dachshunds.
+                </p>
+            </div>
+            <div class="col-md-3">
+                <img src="assets/IMG_7406.jpeg" alt="ellie at graduation">
+            </div>
+        </div>
+        <div class="row section projectSection" id="Projects">
+            <div class="col-md-12">
+                <h3>Projects</h3>
+                <p>Have a look at some of the projects I've completed so far.</p>
+            </div>
+            <div class="row pt-2 d-flex justify-content-around">
+                <div class="col-sm-12 col-lg-6 pt-5">
+                    <div class="browser">
+                        <img src="assets/robot-stores.png" alt="picture of wine collection application">
+                    </div>
+                </div>
+                <div class="col-sm-12 col-lg-6 pt-5 projectInfo text-center">
+                    <h3>E Commerce Store</h3>
+                    <div class="projectDescription">
+                        <p>
+                            Built an e-commerce store web app with object oriented PHP that implements the SOLID principles. Had to extract data
+                            from an API and import into a MySQL database. Worked as a scrum team of 8 to
+                            plan the architecture and estimate tasks.
+                        </p>
+                        <ul class="projectTechList">
+                            <li>PHP</li>
```

Comment:
does also use mySQL / html/css? can see why you haven't put them but they are on the wine one below